File: src/main/java/com/bajdcc/LALR1/interpret/os/task/TKSystem.java
package com.bajdcc.LALR1.interpret.os.task;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【服务】时间
 *
 * @author bajdcc
 */
public class TKSystem implements IOSCodePage {
	@Override
	public String getName() {
		return "/task/system";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/task/TKSystem.kt
package com.bajdcc.LALR1.interpret.os.task

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【服务】时间
 *
 * @author bajdcc
 */
class TKSystem : IOSCodePage {
    override val name: String
        get() = "/task/system"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
